src/Nest/Analysis/Tokenizers/CharGroupTokenizer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	/// <summary>
+	/// The char_group tokenizer breaks text into terms whenever it encounters a character which is in a defined set. It is mostly useful
+	/// for cases where a simple custom tokenization is desired, and the overhead of use of the pattern tokenizer is not acceptable.
+	/// </summary>
+	public interface ICharGroupTokenizer : ITokenizer
+	{
+		/// <summary>
+		/// The maximum token length. If a token is seen that exceeds this length then it is discarded. Defaults to 255.
+		/// </summary>
+		[JsonProperty("tokenize_on_chars")]
+		IEnumerable<string> TokenizeOnCharacters { get; set; }
```

Comment:
specialized type that takes a union of enum and `char`? `string` is no doubt easier to use.